Typo in require path

#### Short description of the problem:
Webpack build fails with this error: build dev failed: Cannot find module '../util/Logger' 

#### What behavior are you expecting?
In dist/webpack/ionic-environment-plugin.js
````
var Logger_1 = require('../util/Logger');
````
should be: 
````
var Logger_1 = require('../util/logger');
````
(notice the lowercase l)

**Steps to reproduce:**
1. Start new project
2. Run ionic serve

**Which @ionic/app-scripts version are you using?**
0.0.41

--- a/src/webpack/ionic-environment-plugin.ts
+++ b/src/webpack/ionic-environment-plugin.ts
@@ -1,5 +1,5 @@
 import { FileCache } from '../util/file-cache';
-import { Logger } from '../util/Logger';
+import { Logger } from '../util/logger';
 import { HybridFileSystem } from './hybrid-file-system';
 import { WatchMemorySystem } from './watch-memory-system';
 

